content/en/logs/log_collection/csharp.md

```diff
@@ -311,6 +311,24 @@ New logs are now directly sent to Datadog.
 [1]: https://www.nuget.org/packages/Serilog.Sinks.Datadog.Logs
 [2]: https://app.datadoghq.com/account/settings#api
 [3]: /logs/#reserved-attributes
+{{% /tab %}}
+{{% tab "Log4Net" %}}
+
+You can manually configure a [RemotingAppender][1] to send logs to the [Datadog API][2] by adding to the web.config this:
+
+```
+<appender name="RemotingAppender" type="log4net.Appender.RemotingAppender" >
```

Comment:
I took a further look at the `RemotingAppender` and I don't think this will work for our use case, since Remoting in .NET means RPCs. Instead we should create our own Appender and distribute it as a library much like https://github.com/loggly/log4net-loggly. That way, when the logs are written, we can write custom logic to send HTTP POST's to the proper logs intake endpoint

Reply:
Thanks!
If this doesn't work, let's close this PR.
I'll see if we should prioritize developping this DatadogAppender.